apis/addonconfigs/hack/boilerplate.go.txt

```diff
@@ -0,0 +1,2 @@
+// Copyright YEAR VMware, Inc. All Rights Reserved.
```

Comment:
question: do we need this file ? Or maybe `YEAR` --> `2022` ?

Reply:
This is the boilerplate template copied from root needed to keep the module standalone. `YEAR` get substituted by Makefile target.